benchmark/_http-benchmarkers.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const child_process = require('child_process');
+
+function AutocannonBenchmarker() {
+  this.name = 'autocannon';
+}
+
+AutocannonBenchmarker.prototype.autocannon_exe = process.platform === 'win32'
```

Comment:
I don't think strings should be put on the prototype. I would just evauluate it in the constructor.